apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -419,16 +467,19 @@ export class WorkshopForm extends React.Component {
             <FormGroup validationState={validation.style.virtual}>
               <ControlLabel>
                 Is this a virtual workshop?
-                <HelpTip>
-                  <p>Please update your selection if/when your plans change.</p>
-                </HelpTip>
+                <HelpTip>{virtualWorkshopHelpTip}</HelpTip>
               </ControlLabel>
               <SelectIsVirtual
                 value={this.currentVirtualStatus()}
                 onChange={this.handleVirtualChange}
                 readOnly={
                   this.props.readOnly ||
-                  VirtualOnlySubjects.includes(this.state.subject)
+                  VirtualOnlySubjects.includes(this.state.subject) ||
+                  this.state.cannotChangeIfWorkshopVirtual
+                }
+                showVirtualOptions={
+                  !!this.props.workshop ||
+                  !this.state.cannotChangeIfWorkshopVirtual
```

Comment:
This is for the edge case if someone is creating a workshop and it is within a month of the start date. In this case, 'in-person' will be the only option to a non-admin.